lib/cartopy/tests/crs/test_mercator.py

```diff
@@ -71,9 +72,36 @@ def test_central_longitude():
 def test_latitude_true_scale():
     lat_ts = 20.0
     crs = ccrs.Mercator(latitude_true_scale=lat_ts)
-    proj4_str = ('+ellps=WGS84 +proj=merc +lon_0=0.0 +lat_ts={} '
-                 '+units=m +no_defs'.format(lat_ts))
+    proj4_str = ('+ellps=WGS84 +proj=merc +lon_0=0.0 +x_0=0.0 +y_0=0.0 '
+                 '+units=m +lat_ts={} +no_defs'.format(lat_ts))
     assert crs.proj4_init == proj4_str
 
     assert_almost_equal(crs.boundary.bounds,
                         [-18836475, -14567718, 18836475, 17639917], decimal=0)
+
+
+def test_easting_northing():
+    false_easting = 1000000
+    false_northing = -2000000
+    crs = ccrs.Mercator(false_easting=false_easting,
+                        false_northing=false_northing)
+    proj4_str = ('+ellps=WGS84 +proj=merc +lon_0=0.0 +x_0={} +y_0={} '
+                 '+units=m +lat_ts=0.0 +no_defs'.format(false_easting,
+                                                        false_northing))
+    assert crs.proj4_init == proj4_str
+
+    assert_almost_equal(crs.boundary.bounds,
+                        [-19037508, -17496571, 21037508, 16764656], decimal=0)
+
+
+def test_scale_factor():
+    # Should be same as lat_ts=20 for a sphere
+    scale_factor = 0.9396926207859084
+    crs = ccrs.Mercator(scale_factor=scale_factor,
+                        globe=ccrs.Globe(ellipse='sphere'))
+    proj4_str = ('+ellps=sphere +proj=merc +lon_0=0.0 +x_0=0.0 +y_0=0.0 '
+                 '+units=m +k_0={} +no_defs'.format(scale_factor))
```

Comment:
`scale_factor` is not formatting consistently on Py 2/3. Might be simpler to just hard-code the result.

Reply:
Will do.